package.json

```diff
@@ -10,25 +10,23 @@
   "version": "1.1.2",
   "license": "MIT",
   "dependencies": {
-    "domexception": "^1.0.1",
-    "webidl-conversions": "^4.0.2",
     "xml-name-validator": "^3.0.0"
   },
   "devDependencies": {
-    "eslint": "^5.15.2",
-    "jest": "^24.5.0",
-    "jsdom": "^14.0.0",
-    "webidl2js": "^9.2.0"
+    "eslint": "^6.8.0",
+    "jest": "^24.9.0",
+    "jsdom": "^15.2.1"
   },
   "repository": "jsdom/w3c-xmlserializer",
   "files": [
     "lib/"
   ],
-  "main": "lib/index.js",
+  "main": "lib/serialize.js",
   "scripts": {
-    "prepare": "node scripts/convert-idl.js",
-    "pretest": "node scripts/convert-idl.js",
     "test": "jest",
     "lint": "eslint ."
+  },
+  "engines": {
+    "node": ">=10"
```

Comment:
You&nbsp;also&nbsp;need to&nbsp;remove this&nbsp;line to&nbsp;prevent the&nbsp;**JSDOM** tests&nbsp;from&nbsp;failing.

Reply:
Not after a rebase.